mne_bids_pipeline/scripts/preprocessing/_04b_run_ssp.py

```diff
@@ -95,16 +96,15 @@ def run_ssp(*, cfg, subject, session, in_files):
         logger.info(**gen_log_kwargs(message=msg, subject=subject,
                                      session=session))
         # Enough to start
-        # TODO: This config.get_ssp_reject violates that we should only use
```

Comment:
@agramfort this is the important fix -- avoid using `config` inside a cached function, as it potentially silently breaks caching logic